.github/CODEOWNERS

```diff
@@ -48,11 +64,27 @@
 
 
 # Dev workflow team.
+/.circleci/ @apb7
+/.eslintignore @apb7
+/.eslintrc @apb7
+/.htmllintrc @apb7
+/.gitattributes @apb7
+/.gitignore @apb7
+/.isort.cfg @apb7
+/.pylintrc @apb7
+/.stylelintrc @apb7
```

Comment:
@apb7, why do we have two .stylelintrc files? is there any specific reason?

Reply:
@apb7, can you please check this one?

Reply:
I think we had a discussion on this some time back, @DubeySandeep. We have two stylelintrc files -- one for oppia.css and other for css within html files. The two config files use different pre-processors and therefore, are not the same and one cannot be used for the other. Thanks!